client/containers/EventsPanelContainer/index.jsx

```diff
@@ -29,9 +29,10 @@ EventsPanel.propTypes = {
 
 const mapStateToProps = (state) => ({
     // selectors.getShowEventDetails is either true, an event id, or null
-    eventToDetail: selectors.getShowEventDetails(state) === true
-        ? DEFAULT_FORM_VALUES
-        : selectors.getEventToBeDetailed(state),
+    eventToDetail: selectors.getShowEventForm(state) ? selectors.getShowEventDetails(state) === true
```

Comment:
When we set the showEventDetail to an event-detail object / true value, it opens the Event form for editing as it is mapped to the EventForm's props. I need the event-detail object for delete modal because I need _plannings. But, I don't want the event form to open. Hence added showEventForm to set the intention that Event form should be open. So event form can depend on that to open itself instead of showEventDetail.